File: ExcelSheetColumnNumber.py
class Solution:
    # @param s, a string
    # @return an integer
    def titleToNumber(self, s):
        res = 0
        for i in s:
            res = res*26 + (ord(i) - ord('A') + 1)
        return res

if __name__ == '__main__':
    so = Solution()
    klist = ['AA', 'A']
    for k in klist:
        print so.titleToNumber(k)

File: CPP/Second/excel_sheet_column_number.cc
#include <iostream>
#include <algorithm>
#include <vector>
using namespace std;

class Solution {
public:
    int titleToNumber(string s) {
        int res = 0;
        for (auto c: s) {
            res = res*26 + (c - 'A' + 1);
        }
        return res;
    }
};




int main(int argc, char *argv[])
{

    return 0;
}
